contentcuration/contentcuration/frontend/shared/data/__tests__/changes.spec.js

```diff
@@ -356,7 +356,7 @@ describe('Change Types Unhappy Paths', () => {
   it('should throw error when Change is instantiated without a source', () => {
     expect(
       () => new Change({ key: '1', table: TABLE_NAMES.CONTENTNODE, type: CHANGE_TYPES.CREATED }),
-    ).toThrow(new ReferenceError('source should be a string, but undefined was passed instead'));
+    ).toThrow(new TypeError('source should be a string, but undefined was passed instead'));
```

Comment:
So this is weird so I wonder if `jest` was a bit loosey-goosey with the type of error it got when checking things out here.

It was two years ago that the errors here were defined and the tests error types were flip-flopped but only now did they fail apparently.